### ISSUE TITLE
notify-send fails when run via /bin/sh -c

### ISSUE THREAD

**Issue by kevinsperrine** [fgrehm/notify-send-http#3]
I'm trying to get notify-send-http working with gulp-notify on my vagrant box to notify the server on the host; however, gulp-notify fails because the notify-send command fails when run via /bin/sh -c. It always throws the 'Invalid number of options error'. I can't find any way to modify the parameters sent to the notify-send command via gulp-notify, so it doesn't look like this is possible until the binary handles being called in this way.

```
[19:51:44] gulp-notify: [Gulp Error] An error occurred.
[19:51:44] gulp-notify: [Error running notifier] Could not send message: Command failed: /bin/sh -c notify-send "Gulp Error" "An error occurred." --icon "/home/vagrant/Projects/website-com/wp/wp-content/themes/website-theme/node_modules/gulp-notify/assets/gulp-error.png" --category ""
```

**Comment by fgrehm** [comment]
Yup, tagged as a bug, can be easily reproduced with `/bin/sh -c notify-send-client ":sadpada:"`.

What is even weirder is that on my Ubuntu 14.04 machine, using plain old `notify-send` I get an error too :confused: 

```
$ /bin/sh -c notify-send ':cry:'
No summary specified.
```

Dunno what to do yet, so any ideas on how we can solve this would be really appreciated. What I can say for sure is that Ruby's [guard](http://guardgem.org/) has been working fine for me over the last ~6 months (it probably does not prefix commands with `/bin/sh`)

**Comment by kevinsperrine** [comment]
I've opened an issue with the developer of gulp-notify, too. The more I look at this, the more I think the issue resides on that side. It doesn't even seem to me that `/bin/sh -c` is the correct option as that option is for reading commands _from a file_ ([See Here](https://www.gnu.org/software/bash/manual/bash.html#Invoking-Bash)) and executing them. Regardless of that, even if the arguments were sent correctly to notify-send the command would still fail as gulp-notify sends more than 2 arguments and the current version of notify-send-http doesn't support that. As a work-around I'm using script to act as a wrapper for notify-send-client so that the extra parameters get dropped.

```
#!/bin/bash
# /usr/bin/notify-send

# send only the first two arguments to notify-send-client as it doesn't support the others yet. 
/usr/bin/notify-send-client "$1" "$2"
```
